contracts/ens/ens.go

```diff
@@ -100,45 +100,45 @@ func ensNode(name string) common.Hash {
 	return crypto.Keccak256Hash(parentNode[:], parentLabel[:])
 }
 
-func (self *ENS) getResolver(node [32]byte) (*contract.PublicResolverSession, error) {
-	resolverAddr, err := self.Resolver(node)
+func (s *ENS) getResolver(node [32]byte) (*contract.PublicResolverSession, error) {
```

Comment:
Please swap `s` to `ens`. ENS is already an abbreviation, let's not abbreviate it further :)